components/aws/sagemaker/batch_transform/component.yaml

```diff
@@ -126,5 +126,5 @@ implementation:
       --instance_count, {inputValue: instance_count},
       --resource_encryption_key, {inputValue: resource_encryption_key},
       --tags, {inputValue: tags},
-      --output_location_file, {outputPath: output_location}
+      --output_location_output_path, {outputPath: output_location}
```

Comment:
`output_location_output_path` is a confusing param name IMO

Reply:
How come batch_transform is already using the updated format while other components are not ?

Reply:
I'm not sure why, it was the only one that was. What would you suggest I rename this parameter to?

Reply:
anything that does not have `output` twice in it :D

Reply:
Okay. All the other arguments are called `<output_name>_output_path`, though. So while I agree it is not a great name, we would have to change the name of the output itself which would be backwards breaking?

Reply:
Let's keep it as is in that case